README.md

```diff
@@ -203,7 +203,7 @@ Install Starship using any of the following package managers:
 Install the latest version for your system:
 
 ```sh
-sh -c "$(curl -fsSL https://starship.rs/install.sh)"
+curl -sS https://starship.rs/install.sh | sh
```

Comment:
You also need to update L235 of this file, `docs/README.md`, `docs/installing/README.md` and `docs/faq/README.md`